implementation/.github/workflows/label-pr.yml

```diff
@@ -1,17 +1,31 @@
-name: Auto-label PR
+name: Set / Validate PR Labels
 on:
   pull_request:
     branches:
     - main
-
-concurrency: pr_labels
```

Comment:
I think that it might make sense to keep some form of concurrency group for safety reasons as this does run the risk to create a loop limiting the damage of that would be great